pkg/tempopb/tempo.pb.go

```diff
@@ -6,14 +6,15 @@ package tempopb
 import (
 	context "context"
 	fmt "fmt"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+
```

Comment:
Was this change generated by `protoc-gen-gogo`? If not I suggest we leave this out.